packages/charts/examples/components/demo/chart-types/gauge/visual-test/gauge-visual-test.component.ts

```diff
@@ -14,15 +14,15 @@ export class GaugeVisualTestComponent {
         return {
             value,
             max: 200,
-            thresholds: GaugeUtil.createStandardThresholdConfigs(100, 158),
+            thresholds: GaugeUtil.createStandardThresholdsConfig(100, 158),
         };
     }
 
     public onWarningEnabledChange(enabled: boolean): void {
         this.warningEnabled = enabled;
         this.gaugeConfigs = this.gaugeConfigs.map(c => {
             const config = cloneDeep(c);
-            (config.thresholds as IGaugeThresholdConfigs)[StandardGaugeThresholdId.Warning].enabled = this.warningEnabled;
+            (config.thresholds?.definitions as GaugeThresholdDefs)[StandardGaugeThresholdId.Warning].enabled = this.warningEnabled;
```

Comment:
If `definitions` is optional, it is possible that `enabled` has to be optional as well? We may not be able to read `enabled` in case optionally chained property will turn into `undefined`

Reply:
For the purposes of the test component, we know that `definitions` is defined, so for expedience and brevity I cast it as `GaugeThresholdDefs`.